Src/CommandPool.cs

```diff
@@ -85,12 +85,15 @@ public override void Dispose()
             base.Dispose();
         }
 
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
```

Comment:
1. Can you describe a use case where not having this attribute is an issue? I'm just trying to get a better overview of the problem here.

2. Why is the attribute applied only to those couple of delegate types in `AllocationCallbacks`, `CommandPool` and `Device` classes?

Reply:
1) I've added that as Mono Vulkan C# (https://github.com/mono/VulkanSharp/blob/master/src/Vulkan/Interop/ImportedCommands.cs), dunno what is the default calling convension.
2) Didn't had time to add on rest of delegates.

Reply:
Alright. Before I could merge these kind of changes, I would need to get a clear understanding of what problem it solves. I wouldn't just blindly follow other implementations without reasoning. Also, if it's going into master branch, it shouldn't be just a partial fix, but should be applied everywhere.

Could you remove those attributes from this PR and I'd happily merge rest of the changes. We can tackle the `UnmanagedFunctionPointer` in a separate issue or PR if you're interested.

Thanks!

Reply:
Okay, will do ASAP